docs/migration/v8-to-v9.md

```diff
@@ -128,7 +126,7 @@ Older Typescript versions _may_ still work, but we will not test them anymore an
 
      The passed instrumentation instance will override the default instrumentation instance the integration would use, while the `prismaIntegration` will still ensure data compatibility for the various Prisma versions.
 
-  1. Depending on your Prisma version (prior to Prisma version 6), add `previewFeatures = ["tracing"]` to the client generator block of your Prisma schema:
+  3. Depending on your Prisma version (prior to Prisma version 6), add `previewFeatures = ["tracing"]` to the client generator block of your Prisma schema:
```

Comment:
This was intentional because it is rendered in markdown correctly and it is very annoying having to update all of the numbers when you update something in the middle.